src/components/sidebar-menu/SidebarMenu.tsx

```diff
@@ -0,0 +1,41 @@
+import {
+  ListItem,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+  List,
+  SxProps,
+  Theme
```

Comment:
import Each component directly to reduce bundle size

Reply:
Ok, done